code/modules/modular_computers/file_system/programs/nttransfer.dm

```diff
@@ -170,7 +170,7 @@ var/global/nttransfer_uid = 0
 		data["uploading"] = 1
 		data["upload_uid"] = unique_token
 		data["upload_clients"] = connected_clients.len
-		data["upload_haspassword"] = server_password ? 1 : 0
+		data["haspassword"] = server_password ? 1 : 0
```

Comment:
Feel free to. I just decided to never touch the TG UI, and I would stick to this decision at least until refactor part C.